src/neo/Ledger/MemoryPool.cs

```diff
@@ -368,8 +367,7 @@ internal void UpdatePoolForBlockPersisted(Block block, DataCache snapshot)
             if (block.Index > 0 && _system.HeaderCache.Count > 0)
                 return;
 
-            uint _maxTxPerBlock = NativeContract.Policy.GetMaxTransactionsPerBlock(snapshot);
-            ReverifyTransactions(_sortedTransactions, _unverifiedSortedTransactions, (int)_maxTxPerBlock, MaxMillisecondsToReverifyTx, snapshot);
+            ReverifyTransactions(_sortedTransactions, _unverifiedSortedTransactions, (int)ProtocolSettings.Default.MaxTransactionsPerBlock, MaxMillisecondsToReverifyTx, snapshot);
```

Comment:
Should we eliminate the use of `ProtocolSettings.Default` here?

Reply:
It should be fixed by #2337